main/guides/getting-started/start-a-project.md

```diff
@@ -96,32 +96,36 @@ cd ui && yarn start
 
 Leave this running in its own terminal window and visit [http://localhost:3000](http://localhost:3000) in a web browser.
 
-Once here, you will be asked to enable the dapp in your Agoric wallet.
+## Connect the Dapp to the Agoric Wallet
 
-![dapp fungible faucet ui - needs approval](./assets/agoric-ui-dapp-needs-approval.png)
+1. Once here, you will be asked to enable the dapp in your Agoric wallet.
 
-## Connect the Dapp to the Agoric Wallet
+![dapp card store ui - needs approval](./assets/must-enable-dapp.png)
+
+1. Navigate back to [http://127.0.0.1:8000](http://127.0.0.1:8000) and accept the Dapp's request to connect to your wallet.
```

Comment:
We should be consistent in how we capitalize "dapp". I'm in favor of lowercase, but an argument could be made to capitalize, I suppose. (If it's good enough for the Germans, who are we to argue?)